### PR TITLE
Bug 0806460

### PR DESCRIPTION
**Pull request by cs-tushar-kanade**
https://mantis.fortinet.com/bug_view_page.php?bug_id=0806460
https://mantis.fortinet.com/bug_view_page.php?bug_id=0806506

### CODE REVIEW

File: csv-data-management/playbooks/playbooks.json [hunk 1]
```diff
@@ -3,16 +3,16 @@
   "data": [
     {
       "@type": "WorkflowCollection",
-      "name": "CSV Data Management",
-      "description": null,
+      "name": "Sample - CSV Data Management - 1.0.0",
+      "description": "Sample playbooks for \"CSV Data Management\" connector. If you are planning to use any of the sample playbooks in your environment, ensure that you clone those playbooks and move them to a different collection, since the sample playbook collection gets deleted during connector upgrade and delete.",
```
Comment (cs-suyog-jinturkar) [inline csv-data-management/playbooks/playbooks.json:7]: Recommendation: If you are planning to use/modify any of the sample playbook, clone respective playbook and move them to a different collection. Sample playbook collection gets deleted during connector upgrade and uninstall.

File: csv-data-management/playbooks/playbooks.json [hunk 2]
```diff
@@ -501,11 +501,110 @@
...
+          "owners": [],
+          "isPrivate": false,
+          "deletedAt": null,
+          "importedBy": [],
+          "recordTags": []
+        },
+        {
+          "@type": "Workflow",
+          "triggerLimit": null,
+          "name": "Read two CSV and join in single data set",
```
Comment (cs-suyog-jinturkar) [inline csv-data-management/playbooks/playbooks.json]: Read two CSVs and join into single data set

File: csv-data-management/playbooks/playbooks.json (hunk 1 above)
Comment (cs-tushar-kanade) [reply csv-data-management/playbooks/playbooks.json:7]: This is added by framework not by playbook. No need to change

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added review comment

File: csv-data-management/playbooks/playbooks.json (hunk 2 above)
Comment (cs-tushar-kanade) [reply csv-data-management/playbooks/playbooks.json]: updated

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**